dev-server/containers/AppView/index.js

```diff
@@ -6,7 +6,7 @@ import OfflineModeBadge from '../../../src/components/OfflineModeBadge';
 import WebphoneBadge from '../../../src/components/WebphoneBadge';
 import Environment from '../../../src/components/Environment';
 import withPhone from '../../../src/lib/withPhone';
-
+import 'normalize-css/normalize.css';
```

Comment:
For sass-loader in new version, this import will be broken. Please use `import '~normalize-css/normalize.css';`

Reply:
oooh.. that's how they are dealing with node_modules now? I'll update accordingly.